Basic/ClientDriven/Assets/Shaders/SG_Pickups.shadergraph

```diff
@@ -0,0 +1,4716 @@
+{
```

Comment:
Seeing the following shader warning in the inspector for this shader, as well as SG_SimpleLit & UrpTriplanar:
'UnityMetaVertexPosition': implicit truncation of vector type

Reply:
I wasn't seeing the errors in the inspector on my side for some reason, but I believe I've fixed this issue on these shaders by explicitly truncating those vectors from vector4s to vector3s! please let me know if you still see the issue :)

Reply:
Not seeing them anymore 👍